staging/src/k8s.io/client-go/dynamic/scheme.go

```diff
@@ -35,11 +35,10 @@ func init() {
 	metav1.AddToGroupVersion(parameterScheme, versionV1)
 }
 
-// basicNegotiatedSerializer is used to handle discovery and error handling serialization
-type basicNegotiatedSerializer struct{}
+var ExtraSerializers []func(runtime.ObjectCreater, runtime.ObjectTyper) runtime.SerializerInfo
```

Comment:
Same as above. Global shared access to variables like this is something I'd like to avoid.